airflow/www/views.py

```diff
@@ -837,8 +837,8 @@ def log(self):
         if ti:
             host = ti.hostname
             log_loaded = False
-
-            if socket.getfqdn() == host:
+            
+            if conf.get('webserver', 'FORCE_LOCAL_LOGS') or socket.getfqdn() == host:
```

Comment:
I agree. It would be more straightforward to just do a `os.path.exists(loc)` and void adding an option to the configuration. I imagine I took the convoluted approach thinking there maybe was a reason for that not being the default logic.